src/tests/Microsoft.PowerFx.Interpreter.Tests/ConfigTests.cs

```diff
@@ -424,6 +424,84 @@ private void RecalcEngine_Symbol_CultureInfo6_ThreadProc(CultureInfo culture)
             Assert.Equal("Hello World!", helloWorld);
         }
 
+        [Fact]
+        public void Powerfx_Collect_Without_Serielize()
+        {
+            var result = (BlankValue)Run_Collect_Worflow(false);
+            Assert.Equal("Blank()", result.ToString());
+        }
+
+        [Fact]
+        public void Powerfx_Collect_With_Serielize()
+        {
+            var result = (ErrorValue)Run_Collect_Worflow(true);
+            Assert.Equal("Runtime type mismatch", result.Errors[0].Message);
+        }
+
```

Comment:
Suggestion: merge those two [Fact]s into a single [Theory], and pass the differences as arguments to the method